java/ql/src/semmle/code/java/dataflow/FlowSources.qll

```diff
@@ -318,3 +319,34 @@ class AndroidIntentInput extends DataFlow::Node {
     )
   }
 }
+
+/** Method access to external inputs of `android.content.Intent` or `android.os.BaseBundle` object. */
+class IntentGetExtraMethodAccess extends MethodAccess {
+  IntentGetExtraMethodAccess() {
+    exists(AndroidComponent ac |
+      this.getEnclosingCallable().getDeclaringType() = ac and
+      ac.isExported() and
+      this.getMethod().getName().regexpMatch("get\\w+Extra") and
+      this.getMethod().getDeclaringType() instanceof TypeIntent
```

Comment:
Has conveying taint from the Intent to the Bundle yielded by getExtra gone missing? I'm seeing steps for intent -> getStringExtra and for Bundle -> getString but not for intent -> getExtra -> Bundle -> getString?

Reply:
I've tested all flow combinations including intent -> getExtras -> Bundle -> getString and the query seems to work as expected.

Do I still miss something?